scripts/ci/reborn_pr_test_plan.py

```
@@ -384,6 +384,17 @@ def build_plan(
             qa_evidence_changed = True
             reasons.append("recorded QA evidence changed")
             continue
+        if path.startswith(("tests/reborn_", "tests/e2e/reborn_", "scripts/ci/reborn-")):
+            raise ValueError(f"unmapped Reborn test path: {path}")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)